extensions/amp-story/1.0/amp-story-request-service.js

```diff
@@ -63,7 +66,13 @@ export class AmpStoryRequestService {
 
     if (bookendEl.hasAttribute(BOOKEND_CONFIG_ATTRIBUTE_NAME)) {
       const rawUrl = bookendEl.getAttribute(BOOKEND_CONFIG_ATTRIBUTE_NAME);
-      return this.loadJsonFromAttribute_(rawUrl);
+      let credentials = undefined;
+      if (bookendEl.hasAttribute(BOOKEND_CREDENTIALS_ATTRIBUTE_NAME)) {
+        credentials = bookendEl.getAttribute(
+            BOOKEND_CREDENTIALS_ATTRIBUTE_NAME
+        );
+      }
```

Comment:
I think this could be shortened to `const credentials = bookendEl.getAttribute(BOOKEND_CREDENTIALS_ATTRIBUTE_NAME);` as it will be null if nonexistent. You may have to change the type below.